src/formats/language.c

```diff
@@ -94,6 +94,9 @@ int sd_language_load(sd_language *language, const char *filename) {
         memreader *mr = memreader_open_from_reader(r, len);
         memreader_xor(mr, len & 0xFF);
         memread_buf(mr, language->strings[i].data, len);
+        if(language->strings[i].data[len - 1] == '\n') {
+            language->strings[i].data[len - 1] = '\0';
+        }
```

Comment:
Please apply this fixup in `resources/languages.c` instead of `formats/language.c` so as to not change languagetool behavior.

Reply:
I considered handling linebreak in reader and writer, but decided it wasn't worth it and ended up doing this.